app/javascript/all_neighbors_system_dashboard_stack.js

```diff
@@ -20,19 +20,28 @@ class AllNeighborsSystemDashboardStack {
     this.cohort = (this.countLevel.cohorts || []).filter((d) => d.cohort === this.state.cohort)[0] || {}
     this.householdType = (this.project.household_types || []).filter((d) => d.household_type === this.state.householdType)[0] || {}
     this.demographic = (this.householdType.demographics || this.data.demographics || []).filter((d) => d.demographic === this.state.demographics)[0] || {}
-    this.config = this.project.config || this.homelessnessStatus.config || this.demographic.config || this.cohort.config || this.data.config || {}
-    this.series = this.cohort.series || this.homelessnessStatus.series || this.demographic.series || this.countLevel.series || this.data.series || []
+    this.config = this.project.config || this.homelessnessStatus.config || this.demographic.config || this.cohort.config || this.data.config || this.getActiveConfig()
+    this.series = this.cohort.series || this.homelessnessStatus.series || this.demographic.series || this.countLevel.series || this.data.series || this.getActiveSeries()
   }
 
   test() {
     console.debug(this)
   }
 
+  getActiveConfig() {
+    return {}
+  }
+
+  getActiveSeries() {
+    return {};
+  }
+
   inDateRange(dateString, range) {
     const [year, month, day] = dateString.split('-')
     //ruby date month is 1 based while js date month is 0
     const date = Date.parse(new Date(year, month-1, day))
     const [s, e] = range
+    // console.log('comparing dates', new Date(date), new Date(s), new Date(e))
```

Comment:
Are we wanting to keep the testing comments in the code?

Reply:
Nope, good call.  Thanks.